quarkus-picocli/src/test/java/PicocliProfileTestIT.java

```diff
@@ -15,71 +16,54 @@
 import io.quarkus.ts.qe.services.AgeService;
 import io.quarkus.ts.qe.services.HelloService;
 
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
```

Comment:
is the method order test execution needed?

Reply:
if autostart is used, then yes, it is needed; if autostart is not used, then no

Reply:
well, it is anyway, because you can only restart after it started. and we need restart because it sadly behaves differently then stop and start after itself